packages/features/ee/workflows/api/scheduleEmailReminders.ts

```diff
@@ -102,10 +102,10 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
     where: {
       method: WorkflowMethods.EMAIL,
       scheduled: false,
-      cancelled: false,
       scheduledDate: {
         lte: dayjs().add(72, "hour").toISOString(),
       },
+      OR: [{ cancelled: false }, { cancelled: null }],
```

Comment:
`cancelled` is an optional field, before we we only checked for `cancelled: false` which didn't include `null` values. That's why some workflowReminders weren't scheduled